### DISCUSSION TITLE
ADDRESS and PULL

### DISCUSSION THREAD

**Discussion by adesutherland** [adesutherland/CREXX#189] [Progress Updates]
For info, I am working on these - on the basis that with them we would have something that could actually be useful :-)

For ADDRESS I am lighting up some old code I did https://github.com/adesutherland/rexxobjects/tree/master/shellspawn it needs simplfying (or at least getting rid of the c++ stuff). From memory it was quite powerful, for example, automatically creating a console window when called from a process without a console if the io was not redirected (it made sense at the time!), and call back functions to handle io and stuff like that). I will need to get my mac mini working - I am hoping the linux implementation will work on the mac! Anyway if working we will have really good address capabilities with redirects and more.

**Comment by mgrossmann** [comment]
Hello Adrian,

When it comes to ADDRESS, we should have the extensibility aspect in mind right from the start. 
This means the possibility to add own host environments.

Mike
